_includes/contact_form.html

```diff
@@ -351,140 +351,131 @@
-      <label class="bold" for="email">{% t contact_page.support_request_form.labels.email %}</label>
-      <input class="field string sm-col-12" id="email" maxlength="80" name="email" type="text"
-        placeholder="john.smith@example.com"
-        pattern="^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)+$"
-        title="Please enter an email address" required />
-    </div>
-  </div>
+  <!-- START: Topic -->
+  <label class="usa-label" for="00NU0000004z903">{% t contact_page.support_request_form.labels.topic %}</label>
+  <select
+    id="00NU0000004z903"
+    name="00NU0000004z903"
+    required
+    onchange="dynamicdropdown(this.options[this.selectedIndex].value);"
+    class="usa-select"
+  ></select>
+  <!-- END: Topic -->
 
-  <div>
-    <div class="form-group">
-      <label class="bold" for="00NU0000004z90E">{% t contact_page.support_request_form.labels.phone %}</label>
-      <input class="field string sm-col-4 required" id="00NU0000004z90E" maxlength="40" name="00NU0000004z90E"
-        placeholder="5556667777" type="text" pattern="^\d{3}[-. (]*\d{3}[-. (]*\d{4}$"
-        title="Please enter a phone number in 5556667777 format" />
-    </div>
-  </div>
+  <!-- START: Subtopic -->
+  <label class="usa-label" for="sub-topic">{% t contact_page.support_request_form.labels.subtopic %}</label>
+  <select id="sub-topic" name="00NU0000004z905" class="usa-select"></select>
+  <!-- END: Subtopic -->
 
-  <div>
-    <div class="form-group required-field">
-      <label class="bold" for="00NU0000004z903">{% t contact_page.support_request_form.labels.topic %}</label>
-      <select class="block grid-col-8 field select" id="00NU0000004z903" name="00NU0000004z903" title="Issue Topic"
-        required onchange="dynamicdropdown(this.options[this.selectedIndex].value);"></select><br />
-    </div>
-  </div>
+  <!-- START: Agency Application -->
+  <label class="usa-label" for="00Nt0000000OYA2">
+    {% t contact_page.support_request_form.labels.agency_application %}
+  </label>
+  <select id="00Nt0000000OYA2" name="00Nt0000000OYA2" required class="usa-select">
+    <option value="" selected hidden>
+      {% t contact_page.support_request_form.form_helpers.select %}
+    </option>
+    <optgroup label="{% t contact_page.support_request_form.labels.agency_application_common %}">
+      <option value="USAJOBS">{% t contact_page.support_request_form.agencies.usajobs %}</option>
+      <option value="SBA DLAP">{% t contact_page.support_request_form.agencies.sba_dlap %}</option>
+      <option value="Trusted Traveler Programs (Global Entry/Nexus/Sentri)">
+        {% t contact_page.support_request_form.agencies.trusted_traveler_program %}
+      </option>
+    </optgroup>
+    <optgroup label="{% t contact_page.support_request_form.labels.agency_application_all %}">
+      <option value="Customs and Border Protection Jobs">
+        {% t contact_page.support_request_form.agencies.cbp_jobs %}
+      </option>
```

Comment:
Can we disable the line length check for this file? In a past version, I updated this page sot so that all the `<option>` tags fit on one line each so they were easier to alphabetize ex:

```suggestion
      <option value="Customs and Border Protection Jobs">{% t contact_page.support_request_form.agencies.cbp_jobs %}</option>
```

Reply:
> Can we disable the line length check for this file? In a past version, I updated this page so that all the `<option>` tags fit on one line each so they were easier to alphabetize ex:

Yeah, I can do that.

There's not actually a line length enforcement on this file. I'd just manually run the Prettier formatter over it as a starting point, then edited from there.

Upside: I can restore the formatting.

Downside: Maybe we should want line length or other consistent formatting to be applied and enforced? 🤷 

If we decide we need that, maybe we could extract these options out into a data object so that we can have a bit more control or clarity over how to format it.

Reply:
Yeah if we move all the options to a data file that's probably easiest to maintain. And then we could use liquid `sort` filter to sort, and probably even sort by the translated name if needed instead of just the English name (in case they ever differ)

But that seems like a good follow-up PR so don't worry about blocking this branch on that kind of refactor

Reply:
Reverted back to inline options in 401a426.